src/types/settings.ts

```diff
@@ -43,11 +43,31 @@ export interface SiteAnalyticsSettings {
   ga4: string
 }
 
+export interface SitePasswordSettings {
+  password: string | null
+  isAmplifySite: boolean
+  isStagingPrivatised: boolean
+}
+
 export type SiteSettings = SiteInfo &
   SiteLogoSettings & { colours: SiteColourSettings } & {
     socialMediaContent: SiteSocialMediaSettings
   } & SiteFooterSettings &
-  SiteAnalyticsSettings
+  SiteAnalyticsSettings &
+  SitePasswordSettings
+
+interface BackendPasswordNetlifySettings {
+  isAmplifySite: false
+}
+
+interface BackendPasswordAmplifySettings {
+  password: string
+  isAmplifySite: boolean
```

Comment:
why is this set to `boolean` ah? isnt this always true?

Reply:
This is a distinguishing boolean for whether a site is amplify or not! netlify sites will have this as `false`

Reply:
wait I dont get this ya, isnt this why we should be setting 
```
interface BackendPasswordAmplifySettings {
  password: string
  isAmplifySite: true
}
```
because we know that all password sites are definitely amplify sites?

Reply:
yea it should be a fixed `true` for amplify sites - essentially creating the discriminating union